easybuild/easyconfigs/p/pymatgen/pymatgen-3.5.0-intel-2016.02-GCC-4.9-Python-2.7.11.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "PythonPackage"
+
+name = 'pymatgen'
+version = '3.5.0'
+
+homepage = 'https://pypi.python.org/pypi/pymatgen'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+
+# source package not (yet?) in url PYPI_SOURCE 
+#source_urls = [PYPI_SOURCE]
+source_urls = ['http://pypi.python.org/packages/7a/ae/c1bbe01227514bec203934d608c7319fec173791b3c58afa8e169affb976/']
```

Comment:
just use `PYPI_SOURCE`, I think it's safe to assume this download issue will get resolved?